api/v1/model/src/commonMain/kotlin/PagedResponse.kt

```diff
@@ -50,6 +50,21 @@ data class PagingOptions(
     val sortProperties: List<SortProperty>? = null
 )
 
+@Serializable
+data class PagingData(
+    /** The limit for the number of items to return. */
```

Comment:
Has 0 a special meaning, like "no limit"?

Reply:
No.

Reply:
Should we consider doing so then? Are there use-cases where you definitely want to get all entries at once?

Reply:
But doing so would require to make `limit` a `Long` as well, which you [below](https://github.com/eclipse-apoapsis/ort-server/pull/636#discussion_r1668899703) said to be infeasible (as it would allow requesting more than max int entities). So I guess we should keep it as-is.

Reply:
I don't see a need for that currently, and it could be done by providing a very large limit. The API currently enforces a default limit of 20 if the user does not provide a limit, but we probably should also consider setting an upper bound to prevent that a request can use up all memory (the lists are not streamed).